life_expectancy/cleaning.py

```diff
@@ -0,0 +1,75 @@
+"""Module Assignment 1: EU Life Expectancy"""
+
+import argparse
+import pandas as pd
+from pandas import DataFrame
+
+
+def load_data() -> DataFrame:
+    """
+    Loads `eu_life_expectancy_raw.tsv` DataFrame.
+    :return: DataFrame loaded.
+    """
+    data = pd.read_csv(
+        "life_expectancy/data/eu_life_expectancy_raw.tsv",
+        sep='\t',
+        header=0
+    )
+
+    return data
+
+
+def save_data(expectancy_result: DataFrame) -> None:
+    """
+    Save the resulting DataFrame to `pt_life_expectancy.csv`.
+    :param expectancy_result: DataFrame cleaned to be saved.
+    """
+    expectancy_result.to_csv(
+        'life_expectancy/data/pt_life_expectancy.csv',
+        index=False
+    )
+
+
+def clean_data(expectancy_wide: DataFrame, country: str) -> DataFrame:

```

Comment:
Missing default for country
```suggestion
def clean_data(expectancy_wide: DataFrame, country: str = 'PT') -> DataFrame:

```

Reply:
solved